docs/pages/radiogroup.js

```diff
@@ -0,0 +1,695 @@
+// @flow strict
+import { type Node } from 'react';
+import { RadioGroup } from 'gestalt';
+import GeneratedPropTable from '../components/GeneratedPropTable.js';
+import PageHeader from '../components/PageHeader.js';
+import MainSection from '../components/MainSection.js';
+import Page from '../components/Page.js';
+import { multipledocgen, type DocGen } from '../components/docgen.js';
+
+export default function DocsPage({
+  generatedDocGen,
+}: {|
+  generatedDocGen: {| [string]: DocGen |},
+|}): Node {
+  return (
+    <Page title="RadioGroup">
```

Comment:
nit: the title here and name for PageHeader can be autogen as well (`generatedDocGen.RadioGroup.displayName`)

Reply:
+1